### PR TITLE
[JENKINS-38088] Add rerun to all pipeline runs

### PR DESCRIPTION
**Pull request by imeredith**
# Description

ATH https://github.com/jenkinsci/blueocean-acceptance-test/pull/113

See [JENKINS-38088](https://issues.jenkins-ci.org/browse/JENKINS-38088).

# Submitter checklist
- [ ] Link to JIRA ticket in description, if appropriate.
- [ ] Change is code complete and matches issue description
- [ ] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.
- [ ] Ran Acceptance Test Harness against PR changes.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

@reviewbybees

### CODE REVIEW

**Comment by imeredith** [comment]
Ugh I really messed this up. with npm packages.

File: blueocean-core-js/src/js/components/ReplayButton.jsx [hunk 1]
```diff
@@ -76,16 +75,18 @@ export class ReplayButton extends Component {
         const outerClassNames = outerClass.split(' ');
         const innerButtonClass = outerClassNames.indexOf('icon-button') === -1 ? 'btn inverse' : '';
 
-        const isFailed = isRunFailed(this.props.latestRun);
+        const isFailed = isRunFinished(this.props.latestRun);
```
Comment (kzantow) [inline blueocean-core-js/src/js/components/ReplayButton.jsx]: `isFailed`?

Comment (michaelneale) [inline blueocean-core-js/src/js/components/ReplayButton.jsx]: yeah - probably should be `isFinished`?

**Comment by michaelneale** [comment]
@imeredith any chance you can hold off this to give @sophistifunk a change to get https://github.com/jenkinsci/blueocean-plugin/pull/763 in?

**Comment by sophistifunk** [comment]
Doesn't look like this is likely to conflict with my branch, besides the package/shrinkwrap changes.

**Comment by michaelneale** [comment]
ok

**Comment by michaelneale** [comment]
Nice one LGTM - see my note about keeping things (ATH) clean for SCM roll out though before merging .

**Pull request merged**